lib/datetimeWrapper.js

```diff
@@ -277,6 +277,19 @@ var datetimeWrapper = function(moment) {
       return m && m.isValid();
     },
     /*
+     * Is this a valid date that matches the given date mask
```

Comment:
Maybe add a little more info about use case here - i.e., USE THIS WHEN VALIDATING DATES IN FORMS ;)